src/www/ui/ui-browse.php

```diff
@@ -167,6 +167,24 @@ function ShowItem($Upload, $Item, $Show, $Folder, $uploadtree_tablename)
    */
   private function ShowFolder($folderId)
   {
+    if ($folderId!="") {
+    /* Delete Folder Algorithm */
+    $deleteFolder= "<form method='post'>\n"; // no url = this url
+    $text = _("Select the folder to delete:  ");
+    $deleteFolder.= "<P>$text\n";
+    $deleteFolder.= "<select name='deleteFolder' class='ui-render-select2'>\n";
+    $text = _("select folder");
+    $deleteFolder.= "<option value='' disabled selected>$text</option>\n";
+    $deleteFolder.= FolderListOption(-1, 0, 1, -1, true);
+    $deleteFolder.= "</select>";
+    $text = _("Delete");
+    $deleteFolder.= "&nbsp<input type='submit' onclick='myFunction()' value='$text' class='btn btn-outline-danger' style='font-size: 10px;'><script>
+    function myFunction() {
+      alert('Are you sure you want to delete this upload ?');
```

Comment:
All uploads inside this folder will also get deleted. Are you sure you want to delete this folder ?